openkfw/TruBudget
# 1894 workflow item close fix

### Checklist




### Description

* fixed bug in provisioning that broke workflow item data structure, preventing it from being closed (exchangeRate was missing after WF update in provisioning - it is required property if WF budget is set)

### How to test

1. run app, log in as jxavier
2. Open Amazonas Fund > Furniture (this is where the bug could be reproduced)
3. closing every WF item should work now, should also work everywhere else




Closes #1894
